core/orch_test.go

```diff
@@ -242,14 +250,18 @@ func TestSelectTranscoder(t *testing.T) {
 
 	// assert t1 is selected and t2 drained
 	currentTranscoder = m.selectTranscoder()
-	assert.Equal(currentTranscoder, t1)
+	assert.Equal(t1, currentTranscoder)
+	assert.Equal(1, t1.load)
 	assert.NotNil(m.liveTranscoders[strm])
-	assert.Len(m.remoteTranscoders, 4)
+	assert.Len(m.remoteTranscoders, 2)
 
 	// assert transcoder gets added back to remoteTranscoders if no transcoding error
 	_, err := m.Transcode("", nil)
 	assert.Nil(err)
-	assert.Len(m.remoteTranscoders, 4)
+	assert.Len(m.remoteTranscoders, 2)
+	assert.Equal(1, t1.load)
+	m.completeTranscoders(t1)
+	assert.Equal(0, t1.load)
 }
 
 func TestTranscoderManagerTranscoding(t *testing.T) {
```

Comment:
I think this one is also missing a simple test at the beginning where it attempts to transcode with an empty set of remote transcoders - should return a "No transcoders available" error. (Should have been here earlier before this PR, but would be good to add it anyway.)